integrationtest/v5migrationhelper/orm/src/test/java/org/hibernate/search/test/query/objectloading/mixedhierarchy/School.java

```diff
@@ -6,15 +6,12 @@
  */
 package org.hibernate.search.test.query.objectloading.mixedhierarchy;
 
-import jakarta.persistence.Inheritance;
-import jakarta.persistence.InheritanceType;
 import jakarta.persistence.MappedSuperclass;
 
 /**
  * @author Gunnar Morling
  */
 @MappedSuperclass
-@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
```

Comment:
Apparently, this wasn't suppose to work 🙈 

https://github.com/hibernate/hibernate-orm/commit/d95084a59a76df37ad7e058ac8f30b8729df070b#diff-61c77bc90da115e7c654d89648a3fc7262412c6a3729be59739ef57aaba65e5aR423-R425